Changes

```diff
@@ -599,6 +599,9 @@ OCaml 4.13.0
 - #10454: Check row_more in nondep_type_rec.
   (Leo White, review by Thomas Refis)
 
+- #10468: Correctly pretty print Psig_typesubst.
```

Comment:
You could be more explicit (for non ppx writers or compiler developpers) with something like: 

> Correctly print ` type t := ... ` with `-dsource`.